src/feature/button/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import buttonStyle from "./Button.module.css";
+
+interface ButtonProps {
```

Comment:
Можно просто расширить стандартные свойства кнопки, и тогда такие аттрибуты как onClick и children не нужно укзывать в интерфейсе
```suggestion
interface ButtonProps extends HTMLProps<HTMLButtonElement>
```